ra/ra_test.go

```diff
@@ -2374,10 +2421,18 @@ func TestNewOrderReuseInvalidAuthz(t *testing.T) {
 	test.AssertNotEquals(t, *secondOrder.Id, *order.Id)
 	// It should be status pending
 	test.AssertEquals(t, *secondOrder.Status, "pending")
-	// It should have one authorization
-	test.AssertEquals(t, len(secondOrder.Authorizations), 1)
-	// It should have a different authorization than the first order's now-invalid authorization
-	test.AssertNotEquals(t, secondOrder.Authorizations[0], order.Authorizations[0])
+	if features.Enabled(features.NewAuthorizationSchema) {
+		// It should have one authorization
```

Comment:
There are a lot of new stanzas that verify the number of authorizations. Please factor out these checks into a helper function that checks the feature flag, e.g. `test.AssertEquals(numAuthorizations(secondOrder), 1)`.